providers/messagebird/README.md

```diff
@@ -0,0 +1,91 @@
+# Novu Messagebird Provider
+
+A Messagebird sms provider library for [@novu/node](https://github.com/novuhq/novu)
+
+## Usage
+
+# MessageBird Node.js API Client
```

Comment:
It looks like you just copy pasted the docs for the client, please look at some of the other providers and make this Novu centric.